pkgs/development/python-modules/pxml/default.nix

```
@@ -0,0 +1,43 @@
+{ stdenv
+, pythonAtLeast
+, buildPythonPackage
+, fetchPypi
+, blessings
+, six
+, nose
+, coverage
+}:
+ buildPythonPackage rec {
+  pname = "pxml";
+  version = "0.2.13";
+  disabled = pythonAtLeast "3.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0c9zzfv6ciyf9qm7556wil45xxgykg1cj8isp1b88gimwcb2hxg4";
+  };
+
+  propagatedBuildInputs = [ blessings six ];
+  checkInputs = [ nose coverage ];
+
+  # `python setup.py test` has no skip option??
```

by default buildPythonPacages will call the setuptools test command, but you can also just call the test runner directly
```suggestion
  # `python setup.py test` has no skip option??
```

```
  checkPhase = ''
    nosetests -s ...
  ``;
```

---

done.